src/main/webapp/scripts/sys/isoFipsCountryMapScript.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * CU Generic ISO/FIPS Country modification
+ */
+const ISO_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.isoCountryCode';
+const ISO_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.isoCountry.name.div';
+const FIPS_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.fipsCountryCode';
+const FIPS_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.fipsCountry.name.div';
+const EMPTY_STRING = '';
+
+function onblur_lookupFipsCountry(fipsCountryCodeField) {
+    let primaryKeyValue = ensurePrimaryKeyIsUppercase(fipsCountryCodeField);
+    if (primaryKeyValue === null || primaryKeyValue === '') {
```

Comment:
Due to how boolean checks work in JavaScript, you could potentially write this file's various null/empty checks in the format `!primaryKeyValue` or `!primaryKeyValue || primaryKeyValue === ''`, which might help catch more null/empty-value cases. Sorry for not mentioning this earlier.